src/test-container.ts

```diff
@@ -33,6 +33,7 @@ export interface StartedTestContainer {
   stop(options?: OptionalStopOptions): Promise<StoppedTestContainer>;
   getContainerIpAddress(): Host;
   getMappedPort(port: Port): Port;
+  getName(): ContainerName;
```

Comment:
Is there a use case for exposing this?

Reply:
Just for the sake of testing and for the fact that every field of *InspectResult* is exposed with a getter. Per [my previous comment](https://github.com/testcontainers/testcontainers-node/pull/25/files#r319774038), it may be necessary in cases where we would like to know what the docker engine named the container.